### DISCUSSION TITLE
Nodejs 16.11 FATAL ERROR HandleScope when use worker_threads

### DISCUSSION THREAD

**Discussion by darkterra** [zwave-js/zwave-js#3483] [Q&A]
Hello,

I would like to use two USB dongles on an RPI to manage my various connected objects (Zibgee & Zwave), for that I tried to run the piece of code from the **quickstart.md** file from a *worker* and I get this error:
**FATAL ERROR: v8::HandleScope::CreateHandle() Cannot create a handle without a HandleScope**

Does anyone know how to run zwave-js in a separate thread ?

Best regards :)

**Comment by AlCalzone** [comment]
Doesn't look like this is possible (yet).
https://github.com/serialport/node-serialport/issues/1981 and https://github.com/serialport/node-serialport/issues/1938

You'll need to spawn a separate process if you want to run zwave-js decoupled.
